src/components/Committees/CommitteesSidebar/style.scss

```diff
@@ -0,0 +1,42 @@
+.committeessidebar {
+	position: fixed;
+    background: none !important;
+    left: 0;
+    top: 52%;
+    transition: opacity 3s ease;
+    z-index: 1000;
+    padding: 1rem;
+    transform: translateY(-50%);
+    display: flex;
+    flex-flow: column nowrap;
+    //justify-content: space-around;
+
+	.committee {
+		width: 200px;
+		height: 63px;
```

Comment:
Giving a fixed width to each of the images is a bit tricky; you might notice that each diamond isn't scaled the same (ex the "smaller" names for W and AI are much larger than longer ones like Teach LA or Studio). 

Instead, I would recommend:

1. left-align all the committee images, so the diamonds are stacked on top of each other
2. fix the height of both the image and its link container
3. fix the width of the container for each committee, but not the image itself - this should let the image fill as expected 

A side effect of this approach is that it may help you reduce some classes (ex using `display:block` explicitly, not too many `margin` problems). Let me know if this makes sense, this is a tricky review comment and I can walk it through with you 1-on-1!